qutebrowser/browser/downloads.py

```diff
@@ -710,7 +710,16 @@ def _after_create_parent_question(self,
             last_used_directory = os.path.dirname(self._filename)
 
         log.downloads.debug("Setting filename to {}".format(self._filename))
-        if force_overwrite:
+        conflicting_downloads = self._get_conflicting_downloads()
+        if conflicting_downloads:
+            # There is a download in progress with the same name, ask the user
+            # if it/they should be cancelled.
```

Comment:
nit: You don't need the comment. It is pretty clear with the variable name and message text. Although same goes fire the isfile case below...